web/stories/pages/posts/slow.stories.svelte

```diff
@@ -0,0 +1,34 @@
+<script context="module" lang="ts">
+  import { Story } from '@storybook/addon-svelte-csf'
+
+  import { MOCKED_POSTS } from './mocks.js'
+  import MockedSlow from './slow-mock-env.svelte'
+
+  export const meta = {
+    component: MockedSlow,
+    title: 'Pages/Posts/Slow'
```

Comment:
Let’s have `Pages/Slow` (remove `Posts` step). Right now we group pages only if they have some common folder in URL (like `/feeds/add` or `/settings/profile`).